tox.ini

```diff
@@ -1,21 +1,32 @@
 [tox]
-minversion = 1.6
+minversion = 2.0
```

Comment:
What do we need 2.0 for?

Reply:
mainly for the default behavior change about the execution interruption on commands errors

Reply:
Could you update the PR and the commit message with these details? E.g. I'm not aware of this behavior change. Do we really depend on it?

Reply:
I wouldn't say we depend on it, although the behavior is just safer, I'm just quoting here:
"Note that in tox 2.0, the default behavior of tox with respect to treating errors from commands changed. tox < 2.0 would ignore errors by default. tox >= 2.0 will abort on an error by default, which is safer and more typical of CI and command execution tools, as it doesn’t make sense to run tests if installing some prerequisite failed and it doesn’t make sense to try to deploy if tests failed."
Also if we want to add or override environment variables, the `passenv` option is available starting from 2.0

Reply:
Well, a dependency means that we cannot work with a lower version. E.g. tox will fail with 1.6. If it's just an improvement, we should not raise the minimum IMO.

Reply:
I removed the minversion update as it looks working fine, we'll change that if needed in the future